Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -73,7 +74,7 @@ def get_artifact_data(artifact_relative_path: str) -> Optional[str]:
 def get_fields():
     failed_tests = []
     # failed_tests.txt is copied into the artifacts directory
-    failed_tests_file_path = os.path.join(ARTIFACTS_FOLDER, 'failed_tests.txt')
+    failed_tests_file_path = os.path.join(ARTIFACTS_FOLDER_XSOAR, 'failed_tests.txt')
```

Comment:
There will never be a case when this artifact will be in the other directory?

Reply:
this is the nightly flow which only runs against XSOAR